core/models.py

```diff
@@ -144,7 +144,10 @@ def save(self: "PostVote", *args: int, **kwargs: int) -> None:
         post_votes = PostVote.objects.filter(post=self.post)
         up_votes = post_votes.filter(choice=PostVote.UPVOTE).count()
         down_votes = post_votes.filter(choice=PostVote.DOWNVOTE).count()
-        Post.objects.filter(pk=self.post.pk).update(up_votes=up_votes, down_votes=down_votes)

```

Comment:
Please revert this format change. It's just whitespacing change which  so should not be committed. You may need to fine tune your editor settings.